projects/layout/README.md

```diff
@@ -14,11 +14,15 @@ npm install @studiohyperdrive/ngx-layout
 
 ### 2.1 Configurable layout
 
-The first in the spotlight is a combination of two standalone components related to dynamic views.
+The `configurable layout` provides the ability to render components in a grid depending on a provided two dimensional array of keys and corresponding items with a provided template. The combination exists of an `ngx-configurable-layout` and an `ngx-configurable-layout-item`.
 
-The combination exists of an `<ngx-configurable-layout>` and an `<ngx-configurable-layout-item>` (henceforth named `wrapper` and `item` respectively).
+#### 2.1.1 Setup
 
-The idea of these components is that the order of the items within the template.html does not define the order of the items in the DOM. This is especially useful when building a customizable view. The order in which the items will be rendered, depends on either an input or a reactive form control.
+By using content projection, we render our components inside of a `ngx-configurable-layout-item`. Each item requires a `key` as an input, which will be used to match the provided component with the two dimensional array we provide to the `ngx-configurable-layout` component.
+
+This means that the order of rendering is now no longer depended on how you provide the components in the template, but by the two dimensional array provided to the `ngx-configurable-layout` component. This significantly streamlines the process and allows you to easily refactor existing flows. 
+
+To provide the array to the `ngx-configurable-layout` component, you can either provide it through the `keys` input, or use the layout as a form input form `reactive forms`. 
```

Comment:
... as a form input form `reactive forms`. => ... as a form input from/for `reactive forms`